lib/tools/system-calls.js

```diff
@@ -608,4 +614,21 @@ systemCallMethods.ls = async function (remotePath) {
   }
 };
 
+systemCallMethods.fileSize = async function (remotePath) {
+  try {
+    let files = await this.ls(remotePath, ['-la']);
```

Comment:
this pipe should be handled by android shell rather than exec itself, so should not be a problem. 
But yeah, we the current implementation has its own pros